README.md

```diff
@@ -10,48 +10,52 @@ Recommended:
 
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/honeycombio/intro-to-o11y-go)
 
-Alternative: Clone and run locally. If you use VSCode Devcontainers, this repository is set up for that. Otherwise, we expect golang to be set up.
+Gitpod is a free cloud environment where you can run the example without needing to clone the code or install Python on your machine.
 
-### Start the App
+Or you can clone this repo and run the app locally. If you use VSCode Devcontainers, this repository is set up for that. Otherwise, the app will require golang to be set up.
```

Comment:
```suggestion
You can also clone this repo and run the app locally. 
If you use VSCode Devcontainers, this repository is set up for that. 
Otherwise, the app will require golang to be set up.
```
Is there a specific version of Golang this app requires? Are there installation instructions for Golang we should link to?

Reply:
Thanks for the suggested changes!

Good question on if it requires a specific version of Go. I see the `go.mod` file in the repo lists a minimum version `1.17` so I'll include that in the instructions.